examples/CMakeLists.txt

```diff
@@ -247,6 +253,11 @@ install(FILES
         ${EXAMPLES_DIR}/bend/bend.yml
         ${EXAMPLES_DIR}/bend/bend.out.json
         ${EXAMPLES_DIR}/bend/mc_vs_md.agr
+        ${EXAMPLES_DIR}/dihedral/dihedral.yml
+        ${EXAMPLES_DIR}/dihedral/dihedral.out.json
```

Comment:
`dihedral.out.json` needs to be added with `git add`

Reply:
Fixed with e4dbfd0.